tests/PHPStan/Rules/Properties/data/read-only-property-phpdoc.php

```diff
@@ -0,0 +1,36 @@
+<?php // lint >= 8.1
```

Comment:
btw is the lint info here still needed? I saw the read-only-property.php file is not using it

Reply:
It's for `make lint`. You can investigate if it's excluded or why it doesn't fail on older versions...

BTW this file should definitely have `lint >= 8.0` because it just contains a promoted property :)

Reply:
jfyi I did not actively look, but stumbled over the reason why `read-only-property.php` doesn't have this yesterday. Your first guess was correct, it's excluded https://github.com/phpstan/phpstan-src/blob/1.8.1/Makefile#L40